<launch>
    <node pkg="touchosc_bridge" type="touchosc_bridge.py" name="touchosc_bridge" output="screen">
        <param name="name" value="ROS Teleop"/>
        <param name="port" value="9000" />
        <param name="print_fallback" value="False"/>
        <param name="load_default" value="False"/>
        <rosparam param="handlers">[teleop]</rosparam>
        <rosparam command="load" file="$(find teleop_handler)/cfg/teleop.yaml" />
    </node>
</launch>
